### DISCUSSION TITLE
BUG: Achievements number is not displayed correctly

### DISCUSSION THREAD

**Discussion by schmelto** [community/community#18556] [Other Feature Feedback, Questions, & Ideas]
https://github.com/schmelto?tab=achievements
![image](https://user-images.githubusercontent.com/30869493/173512969-3f627579-e635-4aa4-8d86-0182901b7261.png)
-> the `3x` Pull Shark

![image](https://user-images.githubusercontent.com/30869493/173514140-6fd2118f-2e9f-4603-a17f-0b3f1b6e687e.png)

Microsoft Edge
Version 102.0.1245.39 (Official build) (64-bit)
Screen Resolution: 1242 x 280

**Comment by victordm6** [comment]
![image](https://user-images.githubusercontent.com/71722951/173684037-9c875893-5b5f-47af-8cdc-8fe5087f384e.png)
for me it's normal, there must be a problem with you.

**Reply by schmelto** [reply]
Ok strange I will take a look again later.

**Reply by schmelto** [reply]
Google Chrome is working also on Safari on an iPhone 13 mini
MS Edge is still not working for me
